lib/puppet_forge/v3/base.rb

```diff
@@ -89,11 +89,18 @@ def get_collection(uri_path)
           new_collection(resp)
         end
 
+        # Faraday's Util#escape method will replace a '+' with '%2B' to prevent it being
+        # interpreted as a space. For compatibility with the Forge API, we would like a '+'
+        # to be interpreted as a space so they are changes to spaces here.
+        def unescape_plus(str)
```

Comment:
This seems like an inaccurate name here. Can we call it convert_plus_to_space, or something like that?